### ISSUE TITLE
[Fixed, Read Desc] PAC 3 Objects Lagging / Flashing and Glitching Around

### ISSUE THREAD

**Issue by Sheeplie** [CapsAdmin/pac3#210]
UPDATE: I FIXED IT GUYS, READ BELOW
Listen up, future PAC3 generations, because I know this shit will annoy you one day. To fix it, simply make the model's bone (that it's attached to) the eyes, rather than the head. It's that simple!
----------------------------------------------------------------------------------------------------------------------------------

This was a problem I used to have on a server of mine, and is now happening on a friends, which leads me to believe that it is a common problem for some people.
Basically, in pac3, objects will either lag and glitch around behind you, or flicker in and out of existence as you move around. Usually only other people see these problems, and for you it's fine.
This was very annoying and people really need some answers. Has anybody had this happen or got an idea?

 Add me on Steam if you can let me know of something:
www.steamcommunity.com/id/Sheeplie

**Comment by LUModder** [comment]
Try updating your GPU drivers if you haven't in a while. If that doesn't help, it's a PAC issue with manual bone merging where the bone merged props and ropes have to be set to translucent to prevent "snapping" and movement lag.

**Comment by Sheeplie** [comment]
I tried ticking the translucent box on the hat prop but that didn't seem to fix it.. am I doing this right? Does my tickrate play a role in this?

**Comment by Sheeplie** [comment]
I still need help guys.

**Comment by LUModder** [comment]
Can we get this closed since it's fixed?
